stable/traefik/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: traefik
-version: 1.17.1
-appVersion: 1.4.6
+version: 1.17.2
+appVersion: 1.5.1
```

Comment:
I needed to update the `traefik` version for the HTTP-01 challenge (https://github.com/containous/traefik/pull/2701). What happens to the chart version in this circumstance?

Reply:
I tried your configuration, also change imageTag in values.yaml, from 1.4.6 to 1.5.1. Then I issued a helm upgrade command. First a message 'acme: Could not find solver for: dns-01' followed by a 'Trying to solve HTTP-01' and finally 'Server responded with a certificate.' Apparantly it still tries to use the dns solver.